jenkins/docker/javascript.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:20.04
+
+# install tzdata first to prevent 'geographic area' prompt
+RUN apt-get update >/dev/null \
+	&& apt-get install -y tzdata \
+	&& apt-get install -y git curl
+
+# nodejs
+RUN curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
+	&& apt-get install -y nodejs
+
+# rest dependencies
+RUN apt-get install -y make gcc g++
+
+# add ubuntu user (used by jenkins)
+RUN useradd -ms /bin/bash ubuntu
+
+USER ubuntu
```

Comment:
we'll discuss later, but neither of those two should be needed
(i.e. USER always forces given uid, and we probably want to use `--user` from outside like we did)